### PR TITLE
Add test to create view with INPUT_ATTACMENT usage

### PR DESCRIPTION
**Pull request by Tony-LunarG**
Pertains to change in PR #895

### CODE REVIEW

File: tests/layer_validation_tests.cpp [hunk 1]
```diff
@@ -3356,6 +3356,29 @@ TEST_F(VkLayerTest, InvalidUsageBits)
...
+    ivci.sType = VK_STRUCTURE_TYPE_IMAGE_VIEW_CREATE_INFO;
+    ivci.image = image2.handle();
+    ivci.viewType = VK_IMAGE_VIEW_TYPE_2D;
+    ivci.format = VK_FORMAT_R8G8B8A8_UNORM;
+    ivci.subresourceRange.layerCount = 1;
+    ivci.subresourceRange.baseMipLevel = 0;
+    ivci.subresourceRange.levelCount = 1;
+    ivci.subresourceRange.aspectMask = VK_IMAGE_ASPECT_COLOR_BIT;
+
+    // Create a view with depth / stencil aspect for image with different usage
```
Comment (tobine) [inline tests/layer_validation_tests.cpp]: invalid comment

**Comment by tobine** [comment]
Kill the stale comment and 
LGTM :+1:

File: tests/layer_validation_tests.cpp (hunk 1 above)
Comment (chrisforbes) [inline tests/layer_validation_tests.cpp]: Move to own test (and possibly rename the existing test to better describe what it's trying to exercide)

**Comment by Tony-LunarG** [comment]
I made it its own test and pushed to master

**Pull request closed without merging**